frameworks/hdfs/universe/marathon.json.mustache

```diff
@@ -72,30 +72,30 @@
     "MESOS_HTTP_AUTHENTICATEE": "com_mesosphere_dcos_http_Authenticatee",
     {{/service.service_account_secret}}
 
-    {{#service.tls.enabled}}
-    "TLS_ENABLED": "{{service.tls.enabled}}",
-    "TASKCFG_ALL_TLS_ENABLED": "{{service.tls.enabled}}",
+    {{#service.security.tls.enabled}}
+    "TLS_ENABLED": "{{service.security.tls.enabled}}",
+    "TASKCFG_ALL_TLS_ENABLED": "{{service.security.tls.enabled}}",
```

Comment:
out of scope, but are both environment variables required?

Reply:
See discussion here: https://github.com/mesosphere/dcos-commons/pull/1789/files/e1c17f46f16f407a8276af8ff439821eb1b2c47f#r148108353